guides/common/modules/proc_adding-a-host-to-host-collections.adoc

```diff
@@ -0,0 +1,15 @@
+
+[id="adding-a-host-to-host-collections_{context}"]
+= Adding a host to Host collections
+
+You can add a host to a host collection in the {ProjectWebUI}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the name of the host you want to modify.
+. In the *Host collections* card, click the vertical ellipsis and select *Add host to collections*.
+. Select the host collection.
+. Click *Add*.
+
+.Verification
+* The *Host collections* card under the *Overview* tab now shows the host collection your host belongs to.
```

Comment:
Is the "Verification" section new? cc @Lennonka @adamlazik1

Reply:
@maximiliankolb It's part of the standard CCS template. It's been there for quite some time. The section is used in other files in this repo too guides/common/modules/proc_enabling-capsule-in-UI.adoc:

Reply:
I am not against this at all. It's just that I haven't seen this before. Thanks for explaining.

What is CCS? Are the templates public?